pkg/agent/core/ngt/handler/grpc/handler.go

```diff
@@ -189,7 +189,8 @@ func (s *server) Search(ctx context.Context, req *payload.Search_Request) (res *
 				})
 			log.Debug(err)
 			stat = trace.StatusCodeAborted(err.Error())
-		case errors.Is(err, errors.ErrEmptySearchResult):
+		case errors.Is(err, errors.ErrEmptySearchResult),
+			err == nil && res == nil:
```

Comment:
why do we add this checking?

Reply:
some time calling CGO (internal/core/algo/ngt) returns no error and empty result, so I added and modified it to return NotFound status.
Maybe it's caused by creating indeices.